examples/curb-detection/lifelong_learning_bench/README.md

```diff
@@ -26,7 +26,7 @@ cd /ianvs #One might use another path preferred
 
 mkdir project
 cd project
-git clone https://github.com/kubeedge/ianvs.git   
+git clone -b feature-lifelong-n https://github.com/kubeedge/ianvs.git   
```

Comment:
feature-lifelong-n  is used in the current version. After merged in the coming version, it should be fine with the previous clone.

Reply:
Ok, I agree with that.